devel/swiftlint/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcodeversion 1.0
+
+github.setup        realm swiftlint 0.52.4
+
+name                swiftlint
```

Comment:
`github.setup` will implicitly set `name` & `version`, so you can drop this and the following lines.